Automattic/jetpack
# cli: Add for-mirrors logic to build

#### Changes proposed in this Pull Request:

This is what's needed to use all the parallel build stuff in the GitHub
Action.

#### Jetpack product discussion


PT: p9dueE-4dJ-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy? Particularly the build job?
* There should be minimal changes to the build artifacts. All the composer.jsons should no longer have an empty "repositories" array, other than hashes in composer's installed packages data in the plugins that should be about it.
